styles/summary.less

```diff
@@ -12,6 +12,9 @@
 }
 
 #summary-table-wrapper {
```

Comment:
This is separate concern. But we should avoid using id's for styling. They lead to strong coupling. In the react projects I work on, we have separate style for each component. which is within the component's directory. Webpack takes cares of packing it together etc.